src/sentry/constants.py

```diff
@@ -177,6 +176,14 @@ def get_all_languages():
     'mysql_pwd',
 )
 
+NOT_SCRUBBED_VALUES = set([
```

Comment:
Unclear what these values should all be if we try to cover these. Would we add `None` and `undefined`? What about stuff like `[object Object]`? Just being overly pedantic so we have some rules here of what does become whitelisted and I feel like we should have an explicit, defined rule that we can/should document.